code/game/objects/items/robot/items/storage.dm

```diff
@@ -265,3 +265,33 @@
 	if(istype(atom, /obj/item/ai_module) && !stored) //If an admin wants a borg to upload laws, who am I to stop them? Otherwise, we can hint that it fails
 		to_chat(user, span_warning("This circuit board doesn't seem to have standard robot apparatus pin holes. You're unable to pick it up."))
 	return ..()
+
+/obj/item/borg/apparatus/material
```

Comment:
hey jim

Reply:
i hate existing

Reply:
apparently my previous, still not even published and in progress branch was what this was based off of

why does vsc default to basing off that branch instead of the branch im actually on? no idea!